services/eligible_case_service.py

```diff
@@ -1,16 +1,83 @@
+import logging
 from typing import List
 from models.questionnaire_case_model import QuestionnaireCaseModel
+from cloud_functions.logging import setup_logger
+
+setup_logger()
```

Comment:
This shouldn't be here, it should only get called once during start up, not in included files.

Reply:
Removed and tests still pass